3.10/arangographml-getting-started.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: default
+description: >-
+  Interface to control all resources inside ArangoGraphML in a scriptable manner
+---
+# Getting Started with ArangoGraphML Services and Packages
+
+{{ page.description }}
+{:class="lead"}
+
+ArangoGraphML is a set of services that provide an easy to use and scalable
+interface for graph machine learning. Since all of the orchestration and training
+logic is managed by ArangoGraph, all that is typically required is a
+specification outlining the data to be used to solve a task.
+
+The following is a guide to show how to use the `arangoml` package in order to:
+- Manage projects
+- Featurize data
+- Submit training jobs
+- Evaluate Metrics
+- Generate predictions
+
+{% hint 'tip' %}
+To enable the ArangoGraphML services,
+[get in touch](https://www.arangodb.com/contact/){:target="_blank"}
+with the ArangoDB team. Regular notebooks in ArangoGraph don't include the
+`arangoml` package.
+{% endhint %}
+
+## Specifications
+
+The `arangoml` package requires a properly formed specification. The specifications
+describe the task being performed and the data being used. The ArangoGraphML services
+work closely together, with one task providing inputs to another.
+
+## Import
+
+The `arangoml` package comes pre-loaded with every ArangoGraphML notebook environment.
+
+To start using it, simply import it:
+
+```
+import arangoml
+```
+
+## Projects
+
+Projects are an important reference used throughout the entire ArangoGraphML
+lifecycle. All activities link back to a project. The creation of the project
+is very simple. 
+
+**Create a project**
+
+```
+project = arangoml.projects.create_project({"name":"ArangoGraphML_Project_Name"})
+```
+
+**List projects**
+
+```
+arangoml.projects.list_projects(limit=limit, offset=offset)
+```
+
+**Lookup an existing project**
+
+```
+project = arangoml.projects.get_project_by_name("ArangoGraphML_Project_Name")
+
+# or by id
+project = arangoml.projects.get_project("project_id")
+```
+
+**Update an existing project**
+
+```
+arangoml.projects.update_project(body={'id': 'project_id', 'name': 'Updated_ArangoGraphML_Project_Name'}, project_id='existing_project_id')
+```
+
+**List models associated with project**
+
+```
+arangoml.projects.list_models(project_name=project_name, project_id=project_id, job_id=job_id)
+```
+
+**Delete an existing project**
+
+```
+api_instance.delete_project(project_id)
+```
+
+## Featurization
+
+The featurization specification asks that you input the following:
+- `featurization_name`: A name for the featurization task.
+- `project_name`: The associated project name. 
+	You can use `project.name` here if already created or retrieved as descried above.
+- `graph_name`: The associated graph name that exists within the database.
+- `vertexCollections`: The list of vertex collections to be featurized.
+  - Each collection contains a list of features or document attributes and details on how to featurize them.
```

Comment:
I think we should elaborate more on `details on how to featurize them`, since they may not be familiar with the possible values for `feature_type` and `feature_generator`